ask/commands.py

```diff
@@ -0,0 +1,181 @@
+import sublime
+import sublime_plugin
+import pieces_os_client as pos_client
+from pieces.settings import PiecesSettings
+import re
+from difflib import Differ 
+from .prompts import *
+import mdpopups
+import time
+
+LIST_ITEMS = [
+	("Ask for bugs fix",BUGS_PROMPT),
+	("Ask for possible bugs", POSSIBLE_BUGS_PROMPT),
+	("Write Cleaner code", CLEANER_CODE_PROMPT),
+	("Add a doc string to this function",DOC_STRING_PROMPT)
```

Comment:
"Fix a bug"
"Find a bug"
"Write cleaner code"
"Add comments to the code"

Reply:
Also here you can use the prompt task pipelines for fixing and adding comments to code